File: RukaLjubavi.Api/Services/Implementations/GradService.cs
using AutoMapper;
using RukaLjubavi.Api.Contracts.Requests;
using RukaLjubavi.Api.Database;
using RukaLjubavi.Api.Dto;
using RukaLjubavi.Api.Models;

namespace RukaLjubavi.Api.Services
{
    public class GradService : BaseDataService<Grad, GradDto, GradInsertRequest>, IGradService
    {
        public GradService(IMapper _mapper, RukaLjubaviDbContext _context) : base(_context, _mapper)
        {

        }
    }
}

File: RukaLjubavi.Android/app/src/main/java/com/fit/ba/rukaljubavi/Services/GradService.kt
package com.fit.ba.rukaljubavi.Services

import com.fit.ba.rukaljubavi.Models.Drzava
import com.fit.ba.rukaljubavi.Models.Grad
import retrofit2.Call
import retrofit2.http.GET

interface GradService {
    @GET("Grad")
    fun getAll(): Call<List<Grad>>
}